pkg/cmd/repo/fork/fork.go

```diff
@@ -251,6 +262,8 @@ func forkRun(opts *ForkOptions) error {
 					if err != nil {
 						return err
 					}
+				} else {
+					return fmt.Errorf("value of --remote-name can't already exist: '%s'", remoteName)
```

Comment:
Nit: this error could be more descriptive, e.g. `a git remote named '...' already exists`